server/protocol/qdatastreamprotocol.py

```diff
@@ -24,11 +30,17 @@ def __init__(self, reader: StreamReader, writer: StreamWriter):
         """
         self.reader = reader
         self.writer = writer
+        self.writer.transport.set_write_buffer_limits(high=0)
 
         # drain() cannot be called concurrently by multiple coroutines:
         # http://bugs.python.org/issue29930.
         self._drain_lock = asyncio.Lock()
 
+    @property
```

Comment:
I wish we won't use `@property` at all. Reason - property is hiding some "behavior" which is not cool. Property are used to have the way for backward code compatibility, when you have some variable, but in fact you need logic and you don't want to change api.